_content/essays/karma_geoff.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Karma"
+authors:
+  - geoff
+external_url: "https://www.accesstoinsight.org/lib/authors/thanissaro/karma.html"
+drive_links:
+  - "https://drive.google.com/file/d/1KNwJ3NH-8HoP0huE7aO0UQHVCNBLdXR5/view?usp=sharing"
+formats: [pdf]
+course: view 
+tags:
+  - karma
+  - effort
```

Comment:
Yeah, just one tag for "karma" here is good enough. While most items will get 4 or 5 tags, there are some that even get zero if they're short and really only about one thing 😁